src/charm.py

```diff
@@ -117,7 +123,7 @@ def _command(self) -> str:
         if self._web_config:
             args.append(f"--web.config.file={WEB_CONFIG_PATH}")
 
-        command = [PUSHGATEWAY_BINARY] + args
+        command = [f"/bin/sh -c '{PUSHGATEWAY_BINARY}"] + args + [f"2>&1 | tee {PUSHGATEWAY_LOG}'"]
```

Comment:
perhaps add a comment to explain this (obscure) shell sorcery?

Reply:
Done